### DISCUSSION TITLE
Highlight the diff between the history versions

### DISCUSSION THREAD

**Discussion by victormayworm** [outline/outline#3784] [Ideas]
Hello,

using outline I feel that is necessary to have a way to see the changes made between the versions of a document, like in a diff tool. This is very useful to validate and approve or discard the collaborations made on the document.

**Comment by ReessKennedy** [comment]
+1 Totally agree. I am coming from usking Dokuwiki which is 20+ years old and has really great difference checking log for each page that lets you compare different revisions just like on Wikipedia 

One way you _COULD_ do this is to export all pages and keep them in your own Github repo, and possibly automate the export. But this is really annoying when it seems like it would be a really nice addition to the core Outline offering.

**Comment by guettli** [comment]
Wow, that's a basic feature. It is hard to believe that you can't show a diff between the versions of a document. I am disappointed. BTW, my employer pays for this tool, and we would really love to see that feature soon.
